pkgs/development/python-modules/cartopy/default.nix

```diff
@@ -46,6 +38,9 @@ buildPythonPackage rec {
   disabledTests = [
     "test_nightshade_image"
     "background_img"
+
+    # Tries to create '/homeless-shelter'
+    "test_gridliner_labels_bbox_style"
```

Comment:
Use
```
preCheck = ''
  export HOME=$(mktemp -d)
'';
```

Reply:
Actually, even with with that the test fails because it tries to access a internet resource.

I’ll just remove the inadequate comment and keep the test disabled.